### DISCUSSION TITLE
where are screenshots saved?

### DISCUSSION THREAD

**Discussion by gokhalen** [GLVis/glvis#253] [General]
I can run GLVis on WSL under Windows with an Xserver (VcXsrv) running and I can visualize my solution. But if I take a screen shot using SHIFT+s  it says "Taking snapshot number 1... done" but it doesn't save it in the current directory or anywhere else, as far as I can tell. So, where does GLVis save screenshots?

**Comment by gokhalen** [comment]
It turns out that GLVis did not have permission to write to the directory the files were in. Using sudo helped and the screenshot was written.

**Comment by tzanio** [comment]
In general the screenshots are saved in the directory from where you run `glvis`, not necessarily where the binary is located.
